### PR TITLE
Issue76 cross signing

### PR DESCRIPTION
**Pull request by bitfriend**

### CODE REVIEW

File: native/effektio/Cargo.toml [hunk 1]
```diff
@@ -60,8 +60,13 @@ wasm-bindgen-futures = "0.4.28"
...
+tokio = { version = "1", features = ["rt-multi-thread", "parking_lot", "sync"] }
 
 [lib]
 name = "effektio"
 crate-type = ["cdylib", "staticlib"]
+
+[dev-dependencies]
+tokio = { version = "1", features = ["rt-multi-thread", "macros"] }
+tokio-test = "*"
+zenv = "0.8.0"
```
Comment (gnunicorn) [inline native/effektio/Cargo.toml:72]:
```suggestion
zenv = "0.8.0"
matrix-sdk-crypto = { git = "https://github.com/matrix-org/matrix-rust-sdk/" }

```

File: native/effektio/src/api/client.rs [hunk 2]
```diff
@@ -236,3 +426,310 @@ impl Client {
...
+            events::{
+                room::message::MessageType,
+                AnySyncMessageLikeEvent, AnySyncRoomEvent, AnyToDeviceEvent,
+            },
+            UserId,
+        },
+        store::StateStore,
+        Client as MatrixClient, LoopCtrl, Result as MatrixResult,
+    };
+    #[cfg(feature = "e2e-encryption")]
```
Comment (gnunicorn) [inline native/effektio/src/api/client.rs:446]:
```suggestion
```

File: native/effektio/Cargo.toml (hunk 1 above)
Comment (bitfriend) [reply native/effektio/Cargo.toml:72]: Thank you.
This code is important for me.

**Review by gnunicorn — APPROVED** [review]
rust seems fine superficially. I was thinking we might want to refactor that `start_sync` and not do it in the login anymore to unblock a bunch of ugly pattern that are currently active ... but let's do that later. (also integration test's would be nice... but a later problem, too)

**Comment by gnunicorn** [comment]
superseeded by #158

**Pull request closed without merging**